provisioning/baremetal_config.go

```diff
@@ -79,33 +77,23 @@ func getDeployKernelUrl() *string {
 	return &deployKernelUrl
 }
 
-// TODO(dtantsur): these two can be removed once we no longer have ironic/inspector split
-
 func getIronicEndpoint() *string {
```

Comment:
My comment above claims that this function can be removed to. I wonder if IRONIC_ENDPOINT is actually only used by the removed inspector.

Reply:
Hmm, the variable itself is probably also used by BMO, but I highly doubt this function is used there - BMO cannot access Ironic over localhost.

Reply:
You're right, this function is not being used anywhere. I have removed it now.